utils/versions.py

```diff
@@ -27,15 +27,12 @@ def get_version(collection_id: Union[int, str], uri: str) -> str:
     Returns a version path.
     """
     collection_id = str(collection_id)
-    uri = uri.rstrip('/')
-    uri_parts = uri.lstrip('/').split('/')
+    uri_parts = uri.strip('/').split('/')
     if str(collection_id) not in uri_parts or len(uri_parts) < 2:
         raise Exception(f"Not a valid version path: {uri}, {uri_parts}")
-    rikolti_data_root, relative_path = uri.split(f"{collection_id}/")
-    path_list = relative_path.split('/')
```

Comment:
This isn't quite right, either. `data` needs to be a discrete path part, not just _in_ the path somewhere. `data in 233/vernacular_metadata_v1/mapped_metadata_v1/ == True`. Truncating to `233/vernacular_meta` would be bad. 

So we actually do want to split on `/` and then check if `data` is in the path list. 

Was there something broken with this implementation of `get_version`? It was pretty tricky to implement, so if it's not broken, I don't think it's worth fixing.